pandas/tests/io/formats/test_to_excel.py

```diff
@@ -169,11 +169,13 @@
      {'alignment': {'wrap_text': True}}),
 ])
 def test_css_to_excel(css, expected):
+    pytest.importorskip('cssdecl')
```

Comment:
do you have an assertion for exception message checking in your testing arsenal?